routes/web.php

```diff
@@ -42,6 +42,13 @@
 
     Route::group(['prefix' => 'users', 'as' => 'user.'], function () {
         Route::get('/', [UserController::class, 'index'])->name('users');
+
+        Route::get('/add', [UserController::class, 'add_user_page'])->name('add_user_page');
```

Comment:
edit funtion controller name, use camelCase. Exp:  addUserPage. Do for all case related